src/bulma/pages/components/Order.vue

```diff
@@ -158,7 +158,7 @@ export default {
             const field = this.form.field(this.form.param('fulfillingAttribute'));
 
             field.value = field.value
-                ? null
+                ? ''
                 : format(new Date(), 'Y-m-d H:i:s');
 
```

Comment:
the change above avoids a couple of extra events Datepicker respectively CoreDatepicker events.

When previously setting the value to null from a date, when changing the value we emitted an input event and then 
we called clear() on the underlying Flatpicker instance.

The clear method of the Flatpicker instance was then, setting the input value to an empty string 
`self.input.value = "";`
thus generating a new set of input events.

In the end, the value was going to end up empty string no matter what, with this change we're skipping redundant events.